src/utilities/GameStateManager.cpp

```diff
@@ -58,5 +68,10 @@ bool GameStateManager::canMoveOutsideField(int robotId) {
     }
     return true;
 }
+
+    Vector2 GameStateManager::getRefereeDesignatedPosition() {
+        auto designatedPos = rtt::ai::GameStateManager::getRefereeData().designated_position();
+        return Vector2(designatedPos.x()/1000, designatedPos.x()/1000);
```

Comment:
This is definitely not the way to do it; we should store it correctly within AI IMO.